tests/compute/console_test.go

```diff
@@ -134,7 +117,13 @@ var _ = SIGDescribe("[rfe_id:127][posneg:negative][crit:medium][vendor:cnv-qe@re
 			})
 
 			It("[test_id:1593]should not be connected if scheduled to non-existing host", func() {
-				vmi := libvmi.NewAlpine(withNodeAffinityTo("kubernetes.io/hostname", "nonexistent"))
+				vmi := libvmi.NewAlpine(
+					libvmi.WithNodeAffinityFor(&k8sv1.Node{
+						ObjectMeta: metav1.ObjectMeta{
+							Name: "nonexistent",
+						},
+					}),
+				)
```

Comment:
This trick could be avoided with https://github.com/kubevirt/kubevirt/pull/10973/commits/4a773e5eb9aac52d98cf2ac875d69430ec521136

Reply:
I agree, I can rebase if it gets in before this one.